doc/man3/OSSL_CMP_SRV_CTX_new.pod

```diff
@@ -89,6 +89,10 @@ Its arguments are the B<OSSL_CMP_SRV_CTX> I<srv_ctx> and the CMP request message
 I<req>. It does the typical generic checks on I<req>, calls
 the respective callback function (if present) for more specific processing,
 and then assembles a result message, which may be a CMP error message.
+If after return of the function
```

Comment:
nit: if return result of I<function> is ...

Reply:
> nit: if return result of `I<function>` is ...

This was meant differently; extended the sentence for clarity.